test/features/createTable.ts

```diff
@@ -7,19 +7,30 @@ interface CreateTableConfig {
   ifNotExists?: boolean;
   columnComment?: boolean;
   tableComment?: boolean;
+  dialectDoesntHaveVarchar?: boolean;
 }
 
-export default function supportsCreateTable(
-  format: FormatFn,
-  { orReplace, ifNotExists, columnComment, tableComment }: CreateTableConfig = {}
-) {
+export default function supportsCreateTable(format: FormatFn, cfg: CreateTableConfig = {}) {
   it('formats short CREATE TABLE', () => {
     expect(format('CREATE TABLE tbl (a INT PRIMARY KEY, b TEXT);')).toBe(dedent`
       CREATE TABLE
         tbl (a INT PRIMARY KEY, b TEXT);
     `);
   });
 
+  if (!cfg.dialectDoesntHaveVarchar) {
+    it('formats short CREATE TABLE with lowercase data types', () => {
+      expect(
+        format('CREATE TABLE tbl (a INT PRIMARY KEY, b VARCHAR);', {
+          dataTypeCase: 'lower',
+        })
+      ).toBe(dedent`
+      CREATE TABLE
+        tbl (a int PRIMARY KEY, b varchar);
+    `);
+    });
+  }
```

Comment:
This test looks completely unnecessary. The lowercasing of data types is already tested by the `supportsDataTypeCase()` test.

It's also problematic to execute this test only for dialects that support `VARCHAR` - it's not like lowercasing of data types shouldn't be supported when dialect has no `VARCHAR` support.